### PR TITLE
moving azure quickstart templates

### PR DESCRIPTION
**Pull request by ronnytag233**
will create 3 more PR's for updating file URI's for azuredeploy.json and then for parameters.json and last one for readme.md

### CODE REVIEW

File: azure-quickstart-templates/rancher-cattle-host-scaling-group/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+- Key Vault Rancher API Secret Key Name - The Secret's name where the Rancher API Secret Key is stored.
+
+## Instructions to Deploy
+
+1. Log into Azure https://portal.azure.com
+1. Click Deploy to Azure button below.  (This will open Azure to a Deployment Template page.)
+1. Fill out the form which is driven by the parameters in the azuredeploy.json with the correct values.  If you have questions about what values to use reach out to folks in DevOps Group.
+1. Agree to the EULA by checking the box.
+1. Click the Purchase button. YOU WILL BE CHARGED, SO GET THE OKAY!!!
+1. After this add a new VPN Peering Connection ```rancherse-to-<vmssName>``` to https://portal.azure.com/#resource/subscriptions/6f7b13dc-a3ff-4e9c-878c-edcc79ecfb9b/resourceGroups/Rancher-RG/providers/Microsoft.Network/virtualNetworks/rancherservervnet/peerings
```
Comment (dustinvanbuskirk) [inline azure-quickstart-templates/rancher-cattle-host-scaling-group/README.md]: We should go and remove the URL here

File: azure-quickstart-templates/rancher-server-scaling-group/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Requires a Microsoft Key Vault setup in Azure.
+
+## Instructions to Prepare for Deploy
+
+The Azure Portal requires the following information, please have it ready before clicking the Deploy to Azure button below.
+
+- Subscription - Subscription Name you would like to deploy the Resource Group to.
+- Resource Group - The name of an existing Resource Group or a new one. - rancher-server-rg
+- Location - Azure Location of Resource Group - ex. West US
+- VM Sku - The size of the VMs - default is Standard_DS2_v2, 2 cpu, 7 GB ram, 124GB persistent disk for Docker
```
Comment (dustinvanbuskirk) [inline azure-quickstart-templates/rancher-server-scaling-group/README.md]: 128GB not 124GB

File: azure-quickstart-templates/rancher-cattle-host-scaling-group/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+Every Cattle Host has a mount /mnt/cattlestorage to a persistent Azure Storage File Store (cattlestorage) that is Shared between all Cattle Hosts Resource Group.
+
+## Instructions to Prepare for Deploy
+
+The Azure Portal requires the following information, please have it ready before clicking the Deploy to Azure button below.
+
+- Subscription - Subscription Name you would like to deploy the Resource Group to.
+- Resource Group - The name of an existing Resource Group or a new one. - rancher-cattle-environment-rg
+- Location - Azure Location of Resource Group - ex. West US
+- VM Sku - The size of the VMs - default is Standard_DS2_v2, 2 cpu, 7 GB ram, 124GB persistent disk for Docker
```
Comment (dustinvanbuskirk) [inline azure-quickstart-templates/rancher-cattle-host-scaling-group/README.md]: 128GB not 124GB

Comment (ronnytag233) [inline azure-quickstart-templates/rancher-cattle-host-scaling-group/README.md]: 👍

File: azure-quickstart-templates/rancher-server-scaling-group/README.md (hunk 2 above)
Comment (ronnytag233) [inline azure-quickstart-templates/rancher-server-scaling-group/README.md]: 👍

**Pull request merged**